tests/app/views/contexts/test_section_summary_context.py

```diff
@@ -290,21 +847,24 @@ def test_context_for_driving_question_summary():
             "custom_summary": [
                 {
                     "add_link": "/questionnaire/people/add-person/?return_to=section-summary",
-                    "add_link_text": "Add someone to this household",
+                    "add_link_text": "Add someone to this " "household",
+                    "item_anchor": None,
+                    "item_label": None,
                     "empty_list_text": "There are no householders",
                     "list": {
                         "editable": True,
                         "list_items": [
                             {
-                                "item_title": "Toni Morrison",
-                                "primary_person": False,
                                 "edit_link": "/questionnaire/people/PlwgoG/edit-person/?return_to=section-summary",
-                                "remove_link": "/questionnaire/people/PlwgoG/remove-person/?return_to=section-summary",
+                                "item_title": "Toni " "Morrison",
```

Comment:
Is this a typo? Is each part of the name supposed to be quoted separately? Ive seen it a few times in this PR

Reply:
Shouldn't be quoted separately. In Python that is allowed and just works but shouldn't be as it is confusing.

Reply:
Updated these 👍